Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocControlObject.cs

```diff
@@ -69,39 +72,37 @@ protected IReadOnlyList<string> GetValidationErrors ([NotNull] ElementScope scop
       if (IsValid (scope))
         return new List<string>();
 
-      ElementScope validationErrorsScope;
       string describedBy;
 
       try
       {
-        describedBy = scope["aria-describedby"];
+        describedBy = scope.GetAttributeOrThrow ("aria-describedby");
       }
       catch (MissingHtmlException exception)
       {
         throw new MissingHtmlException (
-            "Could not find validation error field. This could be due to wrong markup or a missing validators.",
+            c_couldNotFindValidationErrorFieldExceptionMessage,
             exception);
       }
 
       var validationErrorIDs = describedBy.Split (new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
 
+      var validationErrorIndex = int.Parse (scope.GetAttributeOrThrow (DiagnosticMetadataAttributes.ValidationErrorIDIndex));
+      var validationErrorsScope = Scope.FindId (validationErrorIDs[validationErrorIndex]);
+
+      // re-motion renders the delimiter as <br />, but the browser (and Selenium) show it as <br>
+      const string validationErrorDelimiter = "<br>";
+
       try
       {
-        var validationErrorIndex = int.Parse (scope[DiagnosticMetadataAttributes.ValidationErrorIDIndex]);
-
-        validationErrorsScope = Scope.FindId (validationErrorIDs[validationErrorIndex]);
+        return validationErrorsScope.InnerHTML.Split (new[] { validationErrorDelimiter }, StringSplitOptions.RemoveEmptyEntries);
       }
       catch (MissingHtmlException exception)
       {
         throw new MissingHtmlException (
-            "Could not find validation error field. This could be due to wrong markup or a missing validators.",
+            c_couldNotFindValidationErrorFieldExceptionMessage,

```

Comment:
Could there possibly be a benefit to adding a private method `GetMissingValidationErrorFieldException` or something along those lines instead of only saving the exception message in a constant?

Reply:
What would be the benefit?

Reply:
We have similar precedents in re-motion and I think the reason might be calling methods with certain names is a little more descriptive than passing constants with certain names, also it's one line shorter.

Reply:
Makes sense, I misunderstood your previous comment.

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/dafe24eb39dd3d5dd0a97ae42a728cd48f7e7479.